### PR TITLE
[public] Merge release/skylab-v3 into develop

### PR DESCRIPTION
**Pull request by climbfuji**
## Description

As the title says. Created separate branch because we want `release/skylab-v3` to stick around until the next release.

This PR shows a few more changes than expected, because `develop` in the public space was first created from `main`. With this PR, the public develop branch is identical to the internal develop branch, except the URLs (`JCSDA` instead of `JCSDA-internal`) and the data repositories (not needed for public, data is coming from gdex).

**To be merged as merge commit, not as squash commit**

## Acceptance Criteria (Definition of Done)

PR merged

## Dependencies

none

## Impact

Release branches need to be merged for all repositories in order for develop to function correctly.

### CODE REVIEW

**Comment by climbfuji** [comment]
@shlyaeva I think this is now also correct

**Review by shlyaeva — COMMENTED** [review]
I see here we already had tags in develop before. Do we want to change to develop branches for develop branch of public bundle as well?

File: README.md [hunk 1]
```diff
@@ -1,5 +1,3 @@
-# Note - as of Nov 7, 2022, the main branch of this repository is frozen. The develop branch is used for all releases and all developments.
-
```
Comment (shlyaeva) [inline README.md:2]: I think this can stay in develop.

Comment (climbfuji) [reply README.md:2]: So far we've only added it to the main/master branches, not the develop branches ...

**Comment by climbfuji** [comment]
> I see here we already had tags in develop before. Do we want to change to develop branches for develop branch of public bundle as well?

The tags were in develop, because there was no develop branch until yesterday. I created develop from main, which had fixed tags.

**Comment by climbfuji** [comment]
See https://github.com/JCSDA/fv3-bundle/ - the message about the main/master branch is not in the default (develop) branch, and develop doesn't use tags in CMakeLists.txt. Which I think is correct.

File: README.md (hunk 1 above)
Comment (shlyaeva) [reply README.md:2]: Makes sense, thank you!

**Review by shlyaeva — APPROVED** [review]

**Review by fabiolrdiniz — APPROVED** [review]
Thanks, @climbfuji!

**Pull request merged**